vibra/interface/model_inputs/acoustic/internal_impedances/perforated_plate_model_inputs.py

```diff
@@ -735,7 +737,7 @@ def remove_all_line_properties_boundind_surface(self, new_surface_ids: list[int]
 
         line_properties = deepcopy(self.properties.line_properties)
         for new_surface_id in new_surface_ids:
-            lines_from_surface = self.mesh.lines_from_surface.get(new_surface_id)
+            lines_from_surface = self.geometry.surfaces_to_curves(new_surface_id)
             if lines_from_surface is None:
                 continue
```

Comment:
Some of these checks are not needed anymore, since surfaces_to_curves will never return None.